src/pages/welcome.astro

```diff
@@ -60,9 +60,11 @@ const description = t("site.description");
     ERROR = 'grant_error',
     REJECTED = 'grant_rejected',
     INVALID = 'grant_invalid',
+    KEY_SUCCESS = 'key_add_success',
+    KEY_ERROR = 'key_add_error',
   }
 
-  type Intent = 'connect' | 'funds'
+  type Intent = 'connect' | 'reconnect' | 'funds' | 'update_budget'
```

Comment:
We need to add messages for `reconnect` and `update_budget` as well.

Reply:
added messages for `update_budget` intent.
for the `reconnect` intent we don't need messages as we reuse the grant we have (we do not ask to accept/reject grant), `KEY_SUCCESS` and `KEY_ERROR` act as messages for reconnect